src/pages/workspaces/workspace/applications/AppLauncher.js

```diff
@@ -69,6 +70,7 @@ const AppLauncher = _.flow(
             ['Running', () => 'Almost ready...'],
             ['Stopping', () => 'Notebook runtime environment is stopping, which takes ~4 minutes. You can restart it after it finishes.'],
             ['Stopped', () => 'Notebook runtime environment is stopped. Start it to edit your notebook or use the terminal.'],
+            ['Reconfiguring', () => 'Notebook runtime environment is updating, please wait.'],
```

Comment:
Tangent: You said `Updating` is a status that we can already get back. However, it looks like we're not handling that case well here. Should we add that case while we're at it?

Reply:
Actually, it looks like there's some incorrect behavior above WRT deciding when to show the iframe. Perhaps it's unlikely since it would involve a spark cluster and RStudio, but still something we should fix.